social_django/fields.py

```diff
@@ -1,35 +1,17 @@
 import json
 import six
-import functools
-
-import django
```

Comment:
`import django` needs to be kept for https://github.com/python-social-auth/social-app-django/pull/173.